File: quests/felwitheb/Elishia_Leafrunner.pl
#BeginFile: Elishia_Leafrunner.pl
#Quest for South Felwithe - Elishia Leafrunner: Jeremy's Letter

sub EVENT_SAY {
  if($text=~/letter back to him/i) {
    quest::say("Thank you. Take this letter and see that he gets it."); #Real text still needed
    quest::summonitem(8970); #Letter to Jeremy
  }
}

sub EVENT_ITEM {
  if(plugin::check_handin(\%itemcount, 8969=>1)) { #Letter to Elishia
    quest::say("Oh, a letter from my grandson? Thank you very much. I haven't been able to talk to him recently. Would you mind taking a [letter back to him] for me?");
    quest::faction(170,20);  #Keepers of the Art
    quest::faction(178,20);  #King Tearis Thex
    quest::faction(99,20);   #Faydarks Champions
    quest::faction(322,-20); #The Dead
  }
  plugin::return_items(\%itemcount);
} 

File: felwitheb/Elishia_Leafrunner.lua
-- items: 8970, 8969
function event_say(e)
	if(e.message:findi("letter back to him")) then
		e.other:SummonItem(8970); -- Letter to Jeremy
	end
end

function event_trade(e)
	local item_lib = require("items");

	if(item_lib.check_turn_in(e.trade, {item1 = 8969})) then -- Letter to Elishia
		e.self:Say("Oh, a letter from my grandson? Thank you very much. I haven't been able to talk to him recently. Would you mind taking a [letter back to him] for me?");
		e.other:Ding();
		e.other:Faction(275,10,0);  -- Keepers of the Art
		e.other:Faction(279,10,0);  -- King Tearis Thex
		e.other:Faction(246,10,0);   -- Faydarks Champions
		e.other:Faction(239,-10,0); -- The Dead
	end
	item_lib.return_items(e.self, e.other, e.trade);
end

-- Elishia_Leafrunner
